config/settings.yml

```diff
@@ -1480,6 +1480,6 @@
       :memory_threshold: 1.gigabytes
       :nice_delta: 1
     :websocket_worker:
-       :connection_pool_size: 8
+       :connection_pool_size: 26
```

Comment:
@jrafanie 5 puma threads, 5 actioncable worker threads, +1 for a possible remote console connection that [almost immediately releases](https://github.com/ManageIQ/manageiq/blob/master/lib/websocket_server.rb#L73) the DB connection. As we were talking with @himdel this is both a dev+prod thing.